core/backend/src/ViewStateHydrater.ts

```diff
@@ -0,0 +1,112 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { CompressedId64Set, Id64, Id64Array, Id64String } from "@itwin/core-bentley";
+import { HydrateViewStateRequestProps, HydrateViewStateResponseProps, IModelError, IModelStatus, ModelProps, QueryRowFormat, SubCategoryResultRow, ViewAttachmentProps, ViewStateLoadProps } from "@itwin/core-common";
+import { Range3d } from "@itwin/core-geometry";
+import { IModelDb } from "./IModelDb";
+
+export class ViewStateHydrater {
+  private _imodel: IModelDb;
+  public constructor(iModel: IModelDb) {
+    this._imodel = iModel;
+  }
+  public async getHydrateResponseProps(options: HydrateViewStateRequestProps): Promise<HydrateViewStateResponseProps> {
+    const response: HydrateViewStateResponseProps = {};
+    const promises = [];
+    if (options.acsId) promises.push(this.handleAcsId(response, options.acsId));
```

Comment:
Please follow our [coding guidelines](https://www.itwinjs.org/learning/guidelines/typescript-coding-guidelines/) to make code easier to read and to debug. See #7. (We need to start using something like prettier).

Reply:
Missed a bunch.

Reply:
Went back through and think I got all the wrong if statements.. I hope